src/drivers/ins/vectornav/VectorNav.cpp

```diff
@@ -328,7 +328,7 @@ void VectorNav::sensorCallback(VnUartPacket *packet)
 			local_position.vxy_max = INFINITY;
 			local_position.vz_max = INFINITY;
 			local_position.hagl_min = INFINITY;
-			local_position.hagl_max = INFINITY;
+			local_position.hagl_max_z = INFINITY;
```

Comment:
either also set `hagl_max_z = INFINITY` or don't set any of those control limits as the struct is initialized with zeros and that the msg definition says "set to 0 when limiting not required".